routes/api/contacts-router.js

```diff
@@ -0,0 +1,77 @@
+import express from "express";
+import contactsService from "../../models/contacts-model.js";
+import Joi from "joi";
+
+import HttpError from "../../helpers/HttpError.js";
+
+const router = express.Router();
+
+const ContactSchema = Joi.object({
+  name: Joi.string().required().messages({ "any.required": "name" }),
+  email: Joi.string().required().messages({ "any.required": "email" }),
+  phone: Joi.string().required().messages({ "any.required": "phone" }),
+});
+
+router.get("/", async (req, res, next) => {
+  try {
+    const result = await contactsService.listContacts();
+    res.json(result);
+  } catch (error) {
+    next(error);
+  }
+});
+
+router.get("/:contactId", async (req, res, next) => {
+  try {
+    const { contactId } = req.params;
+    const result = await contactsService.getContactById(contactId);
+    if (!result) throw HttpError(404, `id=${contactId} not found`);
+
+    res.json(result);
+  } catch (error) {
+    next(error);
```

Comment:
Router functions should be separated in controllers or middlewares. Here they should only be invoked, not defined

Reply:
Controllers are separated now. Also added ctrlWrapper and addContactValidate middlewares